server/chat.ts

```diff
@@ -1720,7 +1721,8 @@ export const Chat = new class {
 			if (typeof entry !== 'function') continue;
 
 			const handlerCode = entry.toString();
-			entry.requiresRoom = /\bthis\.requires?Room\(/.test(handlerCode);
+			entry.requiresRoom = /\bthis\.require?Room\(/.test(handlerCode);
+			entry.roomSpecific = /requireRoom\((?:'|"|`)(.*?)(?:'|"|`).\)/.exec(handlerCode)?.[1];
```

Comment:
Why not just turn `requiresRoom` into `RoomID | null`? I don't see any backwards compatibility issues.

Reply:
Done.